### ISSUE TITLE
[Question] Warning from tensorflow/keras

### ISSUE THREAD

**Issue by AndreaLanfranchi** [serengil/deepface#1166]
Just wondering if you have any hint about how to address this warning I keep gettting when detectors using TF get instantiated

```
WARNING:tensorflow:From C:\Users\Andrea\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\keras\src\losses.py:2976: The name tf.losses.sparse_softmax_cross_entropy is deprecated. Please use tf.compat.v1.losses.sparse_softmax_cross_entropy instead.
```

**Comment by serengil** [comment]
No, I am not getting this warning at all.

**Comment by serengil** [comment]
[Here](https://stackoverflow.com/questions/78016775/how-to-address-warning-that-tf-losses-sparse-softmax-cross-entropy-is-deprecated), upgrading tf is recommended. What is yours version?

**Comment by AndreaLanfranchi** [comment]
Name: tensorflow Version: 2.15.1
Name: keras Version: 2.15.0

**Comment by serengil** [comment]
And you are getting this when you import deepface as `from deepface import DeepFace`, right?

**Comment by AndreaLanfranchi** [comment]
Yes

**Comment by serengil** [comment]
is there any warning when you import tensorflow as `import tensorflow`?

**Comment by AndreaLanfranchi** [comment]
Yes this ones
```
PS D:\> python
Python 3.11.5 (tags/v3.11.5:cce6ba9, Aug 24 2023, 14:38:34) [MSC v.1936 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow
2024-04-05 10:04:29.036092: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
WARNING:tensorflow:From C:\Users\Andrea\AppData\Local\Programs\Python\Python311\Lib\site-packages\keras\src\losses.py:2976: The name tf.losses.sparse_softmax_cross_entropy is deprecated. Please use tf.compat.v1.losses.sparse_softmax_cross_entropy instead.
```

**Comment by serengil** [comment]
Catched! I was suspecting base models or detectors but your tensorflow installation reports this. Cannot find something meaningful about this warning but you may consider to raise a ticket in tensorflow's repo.

**Comment by serengil** [comment]
Closing because warning message is coming from a dependency not from the deepface itself.
